distributedSetup/docker-compose.slave1.yml

```diff
@@ -7,17 +7,14 @@ services:
       - '1883:1883' # MQTT
```

Comment:
is slave1 an appropriate name in our current times?

Reply:
Sure, we can change that. Sugesstions? Partner1/Partner2, but that doesn´t really differentiate between the different roles...

Reply:
does 'mirror' or 'relay' fit their function?

Reply:
how about agent